src/main/java/coydir/ui/PersonCard.java

```diff
@@ -55,9 +55,9 @@ public PersonCard(Person person, int displayedIndex) {
         name.setText(person.getName().fullName);
         employeeId.setText("Employee ID:  " + String.format("%6s", person.getEmployeeId().value).replace(' ', '0'));
         position.setText("Position:  " + person.getPosition().value);
-        phone.setText("Phone number:  " + person.getPhone().value);
-        address.setText("Address:  " + person.getAddress().value);
-        email.setText("Email address:  " + person.getEmail().value);
+        //phone.setText("Phone number:  " + person.getPhone().value);
```

Comment:
Are we keeping it for now?

Technically we could just remove first, then add back if necessary (I think they don't like commented out code).

Reply:
Noted